fenicsxconcrete/experimental_setup/concrete_cylinder.py

```diff
@@ -0,0 +1,248 @@
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
+from fenicsxconcrete.helper import Parameters
+import dolfinx as df
+import numpy as np
+import gmsh
+import os
+import meshio
+import ufl
+from mpi4py import MPI
+
+from fenicsxconcrete.unit_registry import ureg
+from fenicsxconcrete.boundary_conditions.boundary import plane_at, point_at
+from fenicsxconcrete.boundary_conditions.bcs import BoundaryConditions
+
+
+def generate_cylinder_mesh(radius, height, mesh_density):
```

Comment:
I am currently not sure about the style for the docstrings, but could you move the type hints from the docstring into the function header? In this case

```python
def generate_cylinder_mesh(radius : float, height : float, mesh_density : flaot) -> dolfinx.mesh.Mesh:
"""
.....
Parameters
------------
radius:
     radius of the cylinder
"""
```

Reply:
True, I added the type hints where it made sense to me (everywhere in the changes that doesn't accept many different types). For those that accepted a couple of options, I imported `Union` from `typing`, I am not sure if that is going to be the standard or just a workaround.